### PR TITLE
fix: adding types to build [TOL-909]

### PR DESCRIPTION
**Pull request by aodhagan-cf**
Just realised when attempting to use the package that types are not exported 🙃

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "@contentful/live-preview",
-  "version": "1.0.1",
```
Comment (aodhagan-cf) [inline package.json:3]: Github is not automatically doing this, must be something wrong with the setup 🤔

Comment (chrishelgert) [reply package.json:3]: mhh that strange 🤔

Comment (chrishelgert) [reply package.json:3]: we need this plugin for updating the package.json version:
https://github.com/semantic-release/git

Comment (aodhagan-cf) [reply package.json:3]: I think this is already in the package.json

**Review by chrishelgert — APPROVED** [review]

**Pull request merged**